mongodb/src/it/scala/monix/connect/mongodb/MongoSourceSuite.scala

```diff
@@ -342,4 +342,41 @@ class MongoSourceSuite extends AsyncFlatSpec with MonixTaskSpec with Fixture wit
       }
     }
   }
+
+  "findOne" should "find first encountered item that matches the filter" in {
+    val employeeGlen1 = genEmployeeWith(name = Some("Glen"), age = Some(30)).sample.get
+    val employeeGlen2 = genEmployeeWith(name = Some("Glen"), age = Some(40)).sample.get
+    val employeeGlen3 = genEmployeeWith(name = Some("Glen"), age = Some(50)).sample.get
+    val employeeMike1 = genEmployeeWith(name = Some("Mike"), age = Some(30)).sample.get
+    val employeeMike2 = genEmployeeWith(name = Some("Mike"), age = Some(40)).sample.get
```

Comment:
In order to make the test more readable, can we avoid repeating the same name in the variables? 
If the test requires duplicated data,  it would make more sense fore example to duplicate it by the city :)

Reply:
Good point! Committed a change.

Reply:
Thanks, also... setting the age in this case seems irrelevant

Reply:
I'm not setting age anymore :)